src/linux/backend-v4l2.h

```diff
@@ -252,6 +252,26 @@ namespace librealsense
             virtual void acquire_metadata(buffers_mgr & buf_mgr,fd_set &fds, bool compressed_format) = 0;
         };
 
+        // The aim of the kpi checker is to check the frames drops kpi - which requires
+        // that no more than some percentage of the frames are dropped
+        // It is checked using the fps, and the previous corrupted frames, on the last 2 seconds
+        // for example, for frame rate of 30 fps, and kpi of 5%, the criteria will be:
+        // if at least 3 frames (= 30[fps] * 5%[kpi]* 2[sec]) drops have occured in the 2 seconds,
+        // then the kpi is violated
+        class kpi_checker
```

Comment:
Rename to `frame_drop_monitor` - KPI is too generic

Reply:
Done